README.md

```diff
@@ -2,179 +2,86 @@
 
 ## Overview
 
 [Pravega](http://pravega.io) is an open source distributed storage service implementing Streams. It offers Stream as the main primitive for the foundation of reliable storage systems: *a high-performance, durable, elastic, and unlimited append-only byte stream with strict ordering and consistency*.
 
-The Pravega Operator manages Pravega clusters deployed to Kubernetes and automates tasks related to operating a Pravega cluster.
+The Pravega Operator manages Pravega clusters deployed to Kubernetes and automates tasks related to operating a Pravega cluster.The operator itself is built with the [Operator framework](https://github.com/operator-framework/operator-sdk).
+
+### Features
 
 - [x] Create and destroy a Pravega cluster
 - [x] Resize cluster
 - [x] Rolling upgrades
+- [x] Rollback support
+- [x] Support modification of configuration parameters at runtime
+- [x] Open API Schema validation
+- [x] External Access support
+
+### Project status
+
+The project is currently beta. While no breaking API changes are currently planned, we reserve the right to address bugs and change the API before the project is declared stable.
 
-## Requirements
+### Requirements
 
 - Kubernetes 1.15+
 - Helm 3.2.1+
-- An existing Apache Zookeeper 3.6.1 cluster. This can be easily deployed using our [Zookeeper operator](https://github.com/pravega/zookeeper-operator)
-- An existing Apache Bookkeeper 4.9.2 cluster. This can be easily deployed using our [BookKeeper Operator](https://github.com/pravega/bookkeeper-operator)
+- Cert-Manager v0.15.0+ or some other certificate management solution in order to manage the webhook service certificates. This can be easily deployed by referring to [this](https://cert-manager.io/docs/installation/kubernetes/)
+- An Issuer and a Certificate (either self-signed or CA signed) in the same namespace that the Pravega Operator will be installed (refer to [this](https://github.com/pravega/pravega-operator/blob/master/deploy/certificate.yaml) manifest to create a self-signed certificate in the default namespace)
 
 ## Quickstart
 
-We recommend using our [helm charts](charts) for all installation and upgrades (but not for rollbacks at the moment since helm rollbacks are still experimental). The helm charts for pravega operator (version 0.4.5 onwards) and pravega cluster (version 0.5.0 onwards) are published in [https://charts.pravega.io](https://charts.pravega.io/). To add this repository to your Helm repos, use the following command
-```
-helm repo add pravega https://charts.pravega.io
-```
-There are manual deployment, upgrade and rollback options available as well.
+We recommend using our [helm charts](charts) for all installation and upgrades (but not for rollbacks at the moment since helm rollbacks are still experimental). The helm charts for pravega operator (version 0.4.5 onwards) and pravega cluster (version 0.5.0 onwards) are published in [https://charts.pravega.io](https://charts.pravega.io/).
```

Comment:
Lets keep this section very brief... just a 1 liner with pointers to the document for minikube setup and/or setup using the "uber script" should be good....
This is not where we want to mention helm charts or point to documentation regarding the same. That should be done in the "Deploying Pravega" section.